chia/_tests/util/misc.py

```diff
@@ -617,27 +617,6 @@ def marshal(self) -> Dict[str, Any]:
         }
 
 
-T = TypeVar("T")
-
-
-@dataclasses.dataclass(frozen=True)
-class DataTypeProtocol(Protocol):
```

Comment:
I don't know what we use this type for, but it doesn't seem to intersect with the changes you're making. Is this an unrelated clean-up? (if so, it might be good to move it to a separate PR to make it easier to review the main change)

Reply:
circular dependency i had to resolve for the changes in misc